tests/pr_test.go

```diff
@@ -138,12 +137,18 @@ func setupOptions(t *testing.T, prefix string, dir string, terraformVars map[str
 
 func TestRunDACore(t *testing.T) {
```

Comment:
if this is going deploy the manage solution now, I suggest to rename the test since its currently called `TestRunDACore`

Reply:
yes, I was thinking same & wanted to check with you on this function name change..sure..will change it..will address your other comments..thanks